### PR TITLE
Pickle Pint Quantities

### PR DESCRIPTION
**Pull request by noahl**
Install copy_reg handlers to pickle and unpickle Pint objects properly.

### CODE REVIEW

**Comment by noahl** [comment]
This PR fixes an issue where pickled Pint Quantity objects were unpickled pointing to a different `UnitRegistry` than the current one. @cmccoy please take a look.

File: tests/pint_pickle_test.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import pickle
+import unittest
+
+import pint
+
+import perfkitbenchmarker
+
+
+class TestPintPickling(unittest.TestCase):
```
Comment (cmccoy) [inline tests/pint_pickle_test.py]: Nit: whitespace before function definition.

**Comment by cmccoy** [comment]
+1

**Comment by noahl** [comment]
@cmccoy fixed the whitespace issue. PTAL.

**Comment by cmccoy** [comment]
LGTM.

**Pull request merged**